tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -661,7 +740,13 @@ void handleRequestImpl(sourcekitd_object_t ReqObj,
     Opts.SyntacticOnly = SyntacticOnly;
     return Rec(editorOpen(*Name, InputBuf.get(), Opts, Args, std::move(vfsOptions)));
```

Comment:
Existing request handling functions, like this `editorOpen`, will be inlined in this main request handling function in follow-ups.